examples/calendar/backend/privacy_cir/verifier.sol

```diff
@@ -0,0 +1,310 @@
+//
```

Comment:
you have the verifier contract twice in your code?! ... only need one instance

Reply:
Truffle by default checks for contracts folder for compiling / deploying the code. Running `sh setup.sh` and `sh proof.sh` would generate verifier.sol file.